git-extra/git-update

```diff
@@ -5,6 +5,13 @@
 # when confirmation to do so is given.
 
 git_update () {
+	proxy=$(git config --get http.proxy)
+	if test "$proxy" != ""
+	then
+		export https_proxy="http://$proxy"
```

Comment:
From https://git-scm.com/docs/git-config:

> http.proxy::
>   Override the HTTP proxy, normally configured using the 'http_proxy',
>   'https_proxy', and 'all_proxy' environment variables (see `curl(1)`). In
>   addition to the syntax understood by curl, it is possible to specify a
>   proxy string with a user name but no password, in which case git will
>   attempt to acquire one in the same way it does for other credentials. See
>   linkgit:gitcredentials[7] for more information. The syntax thus is
>   '[protocol://][user[:password]@]proxyhost[:port]'. This can be overridden
>   on a per-remote basis; see remote.<name>.proxy

While I think that we cannot replicate the "user name without password triggers credentials to be acquired" idea, I am fairly certain that we should not prefix the value of `http.proxy` with `http://`. Could you delete that prefix, please?